### ISSUE TITLE
Profile: change display of account details page

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/ext-theme#193]
We put email opt out on the user form, but it's sort of awkward:

![image](https://github.com/readthedocs/ext-theme/assets/1140183/cc0875d1-f6df-4032-a54b-56ccda6586fb)

This feels like it's misplaced. On the production instance it's in a form field group for email notification settings, but I'd probably still say this feels like it's placed in the wrong form.

We probably want a dedicated form if we are going to continue this pattern. For now, this form could use the form fieldset at least, though I'm surprised Crispy form handling didn't do this automatically.

cc @humitos

**Comment by humitos** [comment]
Yes, I've done it this way as a temporal workaround during the deprecation of projects without a config file. Note that on the actual dashboard, the layout is displayed properly under a "Email settings" block:

![Screenshot_2023-07-31_15-45-09](https://github.com/readthedocs/ext-theme/assets/244656/b6266a9d-b1b0-4ab4-bfe7-6745054b52ab)

In any case, we want to think more about this and decide how we want to implement it since we talked about using Sendgrid for these groups and maybe this form is not required at all: https://github.com/readthedocs/meta/discussions/68#discussioncomment-6177103

I think this is not a priority right now and we can just delete this form field on September 25th after the deprecation date arrives and come back to discuss how to implement it properly once we require it.

**Comment by agjohnson** [comment]
Sounds good, mostly just raised this start investing time in another pattern if we were going to reuse what we did here. It shouldn't be hard to use the fieldsets through Crispy either way.
